justfile

```
@@ -219,7 +219,7 @@ _dry-run-release: clean deps dart-build
 dry-run-release:
      @CI=true {{just_executable()}} _dry-run-release
 
-compile-android target:
+compile-android target: dart-build
```

That probably fails because we need to run cargo check before in order to run the build scripts. can you please try if this would also work
```suggestion
compile-android target: _codegen-order-workaround
```